### PR TITLE
Note to update GetClrDbg.sh in MIEngine.

### PR DESCRIPTION
**Pull request by rajkumar42**

### CODE REVIEW

**Comment by rajkumar42** [comment]
@gregg-miskelly @caslan

**Comment by gregg-miskelly** [comment]
LGTM

File: src/platform.ts [hunk 1]
```diff
@@ -30,6 +30,7 @@ export function getCurrentPlatform() {
     else if (process.platform === 'linux') {
         // Get the text of /etc/os-release to discover which Linux distribution we're running on.
         // For details: https://www.freedesktop.org/software/systemd/man/os-release.html
+        // When any distro or version is added, please update GetClrDbg.sh in MIEngine
```
Comment (DustinCampbell) [inline src/platform.ts]: Where do we do that?

Comment (gregg-miskelly) [inline src/platform.ts]: https://github.com/Microsoft/MIEngine/blob/master/scripts/GetClrDbg.sh

Do we want to put some text here that makes it clear that we don't necessarily expect community members to do this? (just Microsoft employees)

**Pull request merged**